src/components/createField.jsx

```diff
@@ -373,6 +372,7 @@ export default function connectField(options) {
 
       render() {
         const { props, contextProps } = this
+        if (!contextProps) return null
```

Comment:
Let's use the `warning` here to warn on using formless fields that are currently not supported.

We can use a similar text like:

```js
warning(true, 'Failed to render the field `%s`: expected to be a child of a Form component. Please render fields as children of Form, since formless fields are not currently supported.', props.name)
```

Reply:
Hi @kettanaito,

It should be warning(false,... not warning(true,...
I just copied exact same provided code and I was scratching my head why warning is not getting logged :D